TheWitcher.Domain/Models/HeroesDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TheWitcher.Domain.Models
+{
+    public class HeroesDTO
+    {
+        public int Id { get; set; }
+        public string HeroName { get; set; }
+        public Nullable<int> HeroLevel { get; set; }
```

Comment:
Use int? instead of Nullable<T>.
There is no much impact, but it will be easier to read code.